docs/user-guide/story-setting.md

```diff
@@ -1,6 +1,7 @@
 
  
-The Story Settings panel allows the editor to configure which additional components should appear to end users in the story view (like the Title, a Logo and the Navigation Toolbar): these components, if properly configured, will appear in the top bar of the story itself. The Story settings panel is available in Edit Mode and it can be opened by clicking on the **Settings** button <img src="../img/button/setting-button.jpg" class="ms-docbutton"/>.
+The Story Settings panel allows the editor to configure which additional components should appear to end users in the story view. The Title, a Logo and the Navigation Toolbar, if properly configured, will appear in the top bar of the story itself while the colors of the different components in the story will be chanched by the Theme and the Overlay.
```

Comment:
The Story Settings panel allows the editor to configure which additional components should appear to end users in the story view. The **Title**, a **Logo** and the **Navigation Toolbar**, if properly configured, will appear in the top bar of the story itself while the colors of the different components in the story can be customized in **Default Theme** and **Overlay** section of the story settings.